samcli/local/apigw/local_apigw_service.py

```diff
@@ -405,7 +409,14 @@ def _parse_v1_payload_format_lambda_output(lambda_output: str, binary_types, fla
         body = json_output.get("body")
         if body is None:
             LOG.warning("Lambda returned empty body!")
-        is_base_64_encoded = json_output.get("isBase64Encoded") or False
+
+        # Http API gateway checks lambda response for isBase64Encoded field, and ignore base64Encoded
+        # Rest API gateway checks first the field base64Encoded field, if not exist, it checks isBase64Encoded field
+        is_base_64_encoded = (
+            json_output.get("base64Encoded")
```

Comment:
Do we need to guard against if `base64Encoded` or `isBase64Encoded` is not a`bool`? The if will catch if `base64Encoded` is set to a non `None` object like "true" or "1". Super edge case-y but figured I would at least ask.

Reply:
Done